### ISSUE TITLE
Merge in Drupal 7.x-2.3

### ISSUE THREAD

**Issue by herbdool** [backdrop-contrib/references#14]
Got a commit merging in Drupal 7.x-2.3 https://github.com/backdrop-contrib/references/commit/8c3b54991bd4ed5abb9244d5c2d060864d082633 (meant to do it in a branch but it's in the master branch). Would be helpful to get some people to test before an official release. @jromine would you be able to make a release if it looks good? Or I can make some fixes.

**Comment by jromine** [comment]
It seems like you committed directly without an open issue or submitting a pull request. Is that the normal process?

**Comment by herbdool** [comment]
@jromine no, sorry. I've gotten ahead of myself. I first was thinking I was a maintainer because I saw that I had made some bug fixes before. I lose track of what I maintain 😋

**Comment by herbdool** [comment]
I should clarify that I'm on the Bug Squad which is why I made a couple previous fixes but I didn't follow procedure this time. Are you an active maintainer again?
